css/style.css

```diff
@@ -0,0 +1,48 @@
+#header{
+    display: flex;
+    justify-content: space-between;
+
+    
+}
+
+.nav-link{
+    margin-left: 20px;
+    margin-right: 10px;
+    text-decoration: none;
+}
+
+.nav-link:hover{
+    background-color: violet;
+    border-radius: 15%;
+    padding-left: 15px;
+    padding-right: 15px;
+   
```

Comment:
Please, as recommended among the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), kindly avoid the unnecessary use of blank lines in your code. This offers better readability to your code. This has many occurrences in your code.